posthog/urls.py

```diff
@@ -151,7 +152,7 @@ def social_create_user(strategy, details, backend, user=None, *args, **kwargs):
     team.users.add(user)
```

Comment:
Do not use `team.users` anymore, it's deprecated and too manual anyway. `User.join` is the way.

Reply:
This is not part of this PR though, might be worth fixing in a different one to keep things a bit tidier?

Reply:
Yep!